RecoTauTag/RecoTau/python/tools/runTauIdMVA.py

```diff
@@ -58,6 +88,7 @@ def __init__(self, process, debug = False,
             if discr not in TauIDEmbedder.availableDiscriminators:
                 raise RuntimeError('TauIDEmbedder: discriminator "{}" is not supported'.format(discr))
         self.toKeep = toKeep
+        self.working_points_v2p5 = deepcopy(WORKING_POINTS_v2p5)
```

Comment:
I think it is not necessary that those WPs are a member of the `TauIDEmbedder` class. They can be copied when exactly needed, i.e. in L704

Reply:
removed in [c6f626b](https://github.com/cms-tau-pog/cmssw/pull/165/commits/c6f626bde3fb108a64cda03c5e01dd0e4787b1af)